frontend/src/pages/journal/components/journal-card/journal-card.tsx

```diff
@@ -0,0 +1,25 @@
+import { getValidClassNames } from '#libs/helpers/helpers.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  title: string;
+  onClick: () => void;
+  isActive: boolean;
+};
+
+const JournalCard: React.FC<Properties> = ({ title, onClick, isActive }) => {
+  return (
+    <button
```

Comment:
It's better to use common Button component, just create new `style` there. I think you'll even be able to get rid of this component by passing smth like `isActive ? styles['journal-card-selected'] : styles['journal-card']`